<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LEDIndicatorEdit</class>
 <widget class="QDialog" name="LEDIndicatorEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LEDIndicatorEdit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="m_layoutLED">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_pbAccept">
         <property name="text">
          <string>Accept</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pbCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="m_grpButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="title">
          <string>Select Mode</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QRadioButton" name="m_rbSelectArea">
            <property name="text">
             <string>Select Area</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="m_rbPreview">
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="m_rbEdit">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="m_tlSelectArea">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_grpPageSettings">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="title">
          <string>Page settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Total Pages</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_spPages"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Unit Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_spUnitWidth"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Unit Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_spUnitHeight"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Paging Mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_cbPagingMode"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_grpMsgSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="title">
          <string>Message Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Visual Message</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="m_teVMIMS">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Horizontal Align</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="m_cbHAIMS"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Vertical Align</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="m_cbVAIMS"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="m_pbLoadBitmap">
              <property name="text">
               <string>Load Image</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Font Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFontComboBox" name="m_cbFontIMS"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Size</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="m_spFontSizeIMS"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="m_pbClrBkGnd">
                <property name="text">
                 <string>Background</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_pbClrFrGnd">
                <property name="text">
                 <string>Foreground</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="m_pbSetImage">
              <property name="text">
               <string>Set current Text</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
